packages/analytics/src/models/commands.js

```diff
@@ -1,7 +1,9 @@
 import { Command } from '@linux-odyssey/models'
 
 // eslint-disable-next-line import/prefer-default-export
-export async function errorCommands() {
+export async function errorCommands(pageNumber, itemsPerPage) {
```

Comment:
```suggestion
export async function errorCommands({ pageNumber, itemsPerPage }) {
```

Make them options for flexibility. You need to check the value, if not defined, give a default.